docs/source/bmi.spec.rst

```diff
@@ -115,8 +115,12 @@ grouped by functional category.
    :ref:`get_value`                Get a copy of values of a given variable.
    :ref:`get_value_ptr`            Get a reference to the values of a given variable.
    :ref:`get_value_at_indices`     Get variable values at specific locations.
+   :ref:`get_state`                Get all the variables that together form the state of the model.
+   :ref:`get_state_pr`             Get a reference to the value of the state.
```

Comment:
Might suggest `get_state_ptr` to be consistent with the other functions that get a reference.

Reply:
I see that this is defined as `get_state_ptr` in bmi-python so `get_state_pr` is a typo?